### DISCUSSION TITLE
Show Sort icon in Search result webparts in "List" layout

### DISCUSSION THREAD

**Discussion by vishalgoyal2472** [microsoft-search/pnp-modern-search#4060] [General]
How to show sort icon and sort results with below options in "List" layout
 
<img width="239" alt="image" src="https://github.com/user-attachments/assets/5564b433-587a-4122-8517-ebfdacdbcead">

**Comment by kasperbolarsen** [comment]
Like this, except the "newest/oldest" doesn't make sense as the sort box is showing an up or down arrow indicator for asc/dsc

![image](https://github.com/user-attachments/assets/35f266cb-dcdf-4581-9ce9-dc827eb3102f)
